target/linux/ath79/dts/qca9563_tplink_eap245-v3.dts

```diff
@@ -19,6 +19,7 @@
 		led-failsafe = &led_status_amber;
 		led-running = &led_status_green;
 		led-upgrade = &led_status_amber;
+		label-mac-device = &eth0;
```

Comment:
I noticed other devices also include this alias, so I decided to add it here too. If just adding it here has no real use, feel free to drop this additional patch.

Reply:
This alias has the use that somebody can easily retrieve the MAC address printed on the label via the function get_mac_label:
https://openwrt.org/docs/guide-developer/mac.address#using_the_label_mac_address

This might be helpful e.g. for downstream networks (Freifunk) that want to identify devices in large networks by that.